doc/manual/functions.rst

```diff
@@ -640,6 +640,22 @@ then ``f.(pi,A)`` will return a new array consisting of ``f(pi,a)`` for each
 consisting of ``f(vector1[i],vector2[i])`` for each index ``i``
 (throwing an exception if the vectors have different length).
 
+Moreover, *nested* ``f.(args...)`` calls are *fused* into a single ``broadcast``
+loop.  For example, ``sin.(cos.(X))`` is equivalent to ``broadcast(x -> sin(cos(x)), X)``,
+similar to ``[sin(cos(x)) for x in X]``: there is only a single loop over ``X``,
+and a single array is allocated for the result.   [In contrast, ``sin(cos(X))``
```

Comment:
The usual typographical convention is to use square brackets for parenthetical comments that have nested parentheses.